pytorch_lightning/trainer/connectors/optimizer_connector.py

```diff
@@ -25,7 +27,9 @@ def on_trainer_init(self, enable_pl_optimizer):
         self.trainer.optimizers = []
         self.trainer.optimizer_frequencies = []
 
-    def update_learning_rates(self, interval: str, monitor_metrics=None):
+    def update_learning_rates(
+        self, interval: str, monitor_metrics: Optional[Dict[str, Any]] = None, opt_indices: List[int] = []
```

Comment:
Using a mutable object as the default value is very discouraged: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Use `None` as the default and initialize it to a list at the beginning of the function

Reply:
Thanks for the guide. Will update.